projectfiles/CodeBlocks/libraries.md

```diff
@@ -25,9 +25,9 @@ The other libraries require complicated compilation procedures too in-depth to d
 
 Download and unpack the source of the libraries zlib, libbzip2, boost (version 1.64 or 1.68 preferred
 
--[**Bzip2**](http://www.bzip.org/downloads.html)
--[**Boost**](http://www.boost.org/users/download/)
--[**Zlib**](http://www.zlib.net/)
+- [**Bzip2:**](http://www.bzip.org/downloads.html).
+- [**Boost:**](http://www.boost.org/users/download/).
+- [**Zlib:**](http://www.zlib.net/).
```

Comment:
It doesn't look good that these lines end with colons. Please either remove manual links, i.e. make the lines like

```md
- **Bzip2**: http://www.bzip.org/downloads.html
```

or remove the colons and periods.